.gitignore

```diff
@@ -1,11 +1,14 @@
-# Two-trick pony for OSX and other case insensitive file systems:
-# Ignore ./python binary on Unix but still look into ./Python/ directory.
-/python
-!/Python/
+#####
+# First, rules intended to apply in all subdirectories.
+# These contain no slash, or only a trailing slash.
 
 *.cover
 *.iml
 *.o
+*.a
+*.so*
```

Comment:
fwiw, this is causing a minor inconvenience in packaging for debian as this now matches `debian/README.source`

the original pattern was `libpython*.so*` could that be restored?

Reply:
`debian/README.source` does not exist in the cpython repo, so presumably you can unignore it wherever you're patching it in.  I would think you would want to explicitly unignore anything added to avoid issues like this.

I prefer to keep `*.so*`, because we we don't want to accidentally check in any compiled extension modules (even though that shouldn't happen anyway and any `*.so` other than `libpython*` should theoretically only appear in the ignored `build` directory, ignoring them anyway seems better to me).

However, I could be persuaded to expand that rule to

```
*.so
*.so.*
```

If that works for you @asottile, please submit a PR and ping me.

Reply:
will send a PR :+1: